bot/events/goodbotHandler.js

```diff
@@ -8,26 +8,23 @@ const { Responses } = source('models/responses');
 const { Statistics, EventTypes } = source('models/statistics');
 
 const goodBotRegex = /[Gg][Oo][Oo][Dd]\s+[Bb][Oo][Tt]/;
-const badBotRegex = /[Bb][Aa][Dd]\s+[Bb][Oo][Tt]/;
 
-function botMessage(message, botMode) {
-    const defaultMessage = botMode === 'goodbot' ? ':]' : ':\'[';
+function botMessage(message) {
+    if (message.guild === null || message.author.bot) return;
+    if (!goodBotRegex.test(message.content)) return;
```

Comment:
Same thing here, I would recommend extracting lines 13 and 14 into its own function